tests/driver_cst816s/main.c

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2020 Koen Zandberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the CST816S touch screen driver
+ *
+ * @author      koen Zandberg <koen@bergzand.net>
+ *
+ * @}
+ */
+
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "cst816s_params.h"
+#include "cst816s.h"
+#include "xtimer.h"
+#include "fmt.h"
+#include "thread.h"
+#include "thread_flags.h"
+
+#define CST816S_THREAD_FLAG     (1 << 8)
+#define CST816S_NUM_TOUCHES     5
+
+static void _cb(cst816s_t *dev, void *arg)
+{
+    (void)dev;
+    kernel_pid_t *pid = arg;
+    thread_flags_set((thread_t *)sched_threads[*pid], CST816S_THREAD_FLAG);
+}
+
+static int _dump_cst816s(cst816s_t *dev)
+{
+    puts("Reading data:");
+    cst816s_touch_data_t touches;
+    int num = cst816s_read(dev, &touches);
+    if (num == 0) {
+        printf("Touch at %03u, %03u with %02x\n", touches.x,
+               touches.y, touches.gesture);
```

Comment:
Maybe print a human readable string corresponding to the gesture detected. That would make the application more friendly (01, 02, 03, etc means nothing for me :) ).

Reply:
It should print a string now!